agent/cli/ehrql_telemetry.py

```diff
@@ -0,0 +1,134 @@
+import argparse
+import json
+import os
+import subprocess
+import sys
+from datetime import datetime
+from pathlib import Path
+
+from opentelemetry import context, trace
+
+from agent.executors.local import job_metadata_path
+from common.tracing import setup_default_tracing
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(description="db_telemetry")
```

Comment:
I think it might be worth getting some of the extra detail from the PR description into the help text (or even just the docstring?). The PR is good for breadcrumbs, but I'm just thinking about trying to surface this information more easily the next time someone comes back to this.

Reply:
Good idea, will do.